kotlintest-tests/kotlintest-tests-core/src/test/kotlin/com/sksamuel/kotlintest/EventuallyTest.kt

```diff
@@ -71,6 +72,13 @@ class EventuallyTest : WordSpec() {
         }
         result shouldBe 1
       }
+      "display the underlying assertion failure" {
+        shouldThrow<AssertionError> {
+          eventually(Duration.ofSeconds(2), AssertionError::class.java) {
+            1 shouldBe 2
+          }
+        }.message.shouldEndWith("; underlying cause was expected: 2 but was: 1")
```

Comment:
What would be in this stacktrace?
I think that line 78 should be the one with the stacktrace, but it may be ok to display line 77.

What do you think?

Reply:
I've added 78 to the stack.

Reply:
Seems good